### PR TITLE
BRITECHSRM-138 Filter styling

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

**Comment by thullSL** [comment]
There's an "unassigned" status (0) in the backend which isn't represented here. It wasn't in the old angular version either. Was there a reason for this or has it been overlooked?

**Comment by lclarksl** [comment]
It's not in the UI but should be added in. Thanks

**Comment by lclarksl** [comment]
Added :)

File: app/components/sessionFilterCheckbox.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    svg, svg > rect {
+        width: 15px;
+        height:15px;
+    }
+}
+
+.label {
+    flex-grow: 1;
+    font-family: 'Roboto', sans-serif;
```
Comment (thullSL) [inline app/components/sessionFilterCheckbox.scss]: Roboto in bold looks like a normal font seen through cataracts. Is there any other choices?

**Comment by thullSL** [comment]
:shipit:

**Comment by ianlovell** [comment]
:shipit:

**Pull request merged**